shared/provision/error/container.tsx

```diff
@@ -2,16 +2,26 @@ import RenderError from '.'
 import * as RouteTreeGen from '../../actions/route-tree-gen'
 import {connect} from '../../util/container'
 import openURL from '../../util/open-url'
+import * as AutoresetGen from '../../actions/autoreset-gen'
 
 type OwnProps = {}
 
 export default connect(
-  state => ({error: state.provision.finalError}),
+  state => ({
+    _username: state.provision.username,
+    error: state.provision.finalError,
+  }),
   dispatch => ({
-    onAccountReset: () => openURL('https://keybase.io/#account-reset'),
+    _onAccountReset: (username: string) =>
+      dispatch(AutoresetGen.createStartAccountReset({skipPassword: false, username})),
```

Comment:
It's unfortunate that one has to enter their password again after going through this link. This is not the only flow that suffers from it IIRC. I thought there was a ticket to figure this out but I can't find it now.